auditai/cmh.py

```diff
@@ -0,0 +1,125 @@
+from scipy import stats
+from functools import partial
+import numpy as np
+import pandas as pd
+
+def extract_data(df):
+    # utility function for cmh test and odds ratio
```

Comment:
should have docstrings in keeping with the rest of audit-AI code

Reply:
fixed!